pages/padrinhos/cadastro-padrinho.js

```diff
@@ -0,0 +1,210 @@
+import { useState, useEffect } from 'react';
+import Layout from 'components/Layout/Layout';
+import { getData } from 'functions/getData';
+import FormField from 'components/Blocks/Form/components/FormField';
+import getRecords from 'functions/getRecords';
+
+// TODO: CLEAN CODE!!!
+
+const Padrinhos = (props) => {
+  const [formData, setFormData] = useState({ ...headers });
+  const [status, setStatus] = useState();
+  const [isLoading, setIsLoading] = useState(false);
+  const [imageSrc, setImageSrc] = useState();
+
+  // CHANGE
+  // ===================================
+  const handleOnChange = (e) => {
+    if (e.target.files) {
+      const reader = new FileReader();
+      reader.onload = function (onLoadEvent) {
+        setImageSrc(onLoadEvent.target.result);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+    } else {
+      setFormData({ ...formData, [e.target.id]: e.target.value });
+    }
+  };
+
+  // SUBMIT
+  // ===================================
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    // Form Submit
+    const rawResponse = await fetch('/api/google/submitPadrinho', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ ...formData }),
+    });
+    const content = await rawResponse.json();
+    if (content?.data?.updates?.updatedRows) {
+      setStatus(true);
+    } else {
+      setStatus(false);
+    }
+
+    // Send e-mail
+    const emailResponse = await fetch('https://api.staticforms.xyz/submit', {
+      method: 'POST',
+      body: JSON.stringify({
+        subject: 'Brinca - Cadastro de Padrinhos',
+        honeypot: '',
+        replyTo: '@',
+        accessKey: process.NEXT_PUBLIC_PADRINHOS_FORM_KEY,
```

Comment:
Please use the original form key: NEXT_PUBLIC_FORM_KEY
(Use your key value for your local tests)

Reply:
Done